de.philippun1.Snoop.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "de.philippun1.Snoop",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "omgrep",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--device=dri"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "omgrep",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources": [
+              {
+                "type": "archive",
+                "url": "https://gitlab.gnome.org/philippun1/omgrep/-/archive/0.1/omgrep-0.1.tar.gz",
```

Comment:
onmgrep but the app name is "Snoop"?

Reply:
Snoop is the friendly name, omgrep the repo name (and original name). Seems to give some problems with the appdata file.
Is there a way to make it work? Or should I do a proper rename?

Reply:
the name of the files are important. so yeah.